client/scripts/models/DiscussionDraft.ts

```diff
@@ -0,0 +1,40 @@
+import moment from 'moment-twitter';
+import { IUniqueId } from './interfaces';
+import OffchainAttachment from './OffchainAttachment';
+
+class DiscussionDraft {
+  public readonly id: number;
+  public readonly author: string;
+  public readonly authorChain: string;
+  public readonly title: string;
+  public readonly body: string;
+  public readonly attachments: OffchainAttachment[];
+  public readonly createdAt: moment.Moment;
+  // Draft tags are not created as tag objects yet, so they are just strings
+  public readonly tag: string;
+  public readonly community: string;
+  public readonly chain: string;
+  public readonly slug = 'draft';
```

Comment:
Why do we need this on the object?

Reply:
You're right, I replicated it early in the branch as a precaution, but there's no need now, I'll remove